BackEndApp/authv1/urls.py

```diff
@@ -6,4 +6,7 @@
     path("login/", views.login, name="login"),
     path("register/", views.register, name="register"),
     path("logout/", views.logout, name="logout"),
+    path("password/forgot/", views.forgot_password, name="forgot_password"),
+    path("password/enter/otp/", views.verify_otp, name="verify_otp"),
```

Comment:
do we really need two '/' for /enter/otp? @ADI10HERO

Reply:
Having 2 `/` is not a problem tbh @shintan777  but here the names are misleading. 😞 


@mahekn23, as I repeatedly say, naming is one of the hardest things in software engineering and it's absolutely fine to make mistakes here, having something like. But following consistency with the existing codebase + name the routes/fns what they're doing is important.

Reply:
Really sorry about that. Can you please suggest some names ?